content/SCALE/SCALE22.12.md

```diff
@@ -450,7 +450,8 @@ Additional feature in future Bluefin releases:
 
 | Seen In | Key | Summary | Workaround | Resolved In |
 |---------|-----|---------|------------|-------------|
-| 22.12-BETA.1 | <a href="https://ixsystems.atlassian.net/browse/NAS-117940" target="_blank">NAS-117940</a> | Implements temporary fix for the return from `glfs_open()` to honor `O_DIRECTORY` flag | Pertains to an internal issue in Samba. After gluserfs is fixed with a permanent solution to this issue happens, this temporary fix will be reverted. | Targeted 22.12 |
+| 22.12-BETA.1  |  | Upgrading from 22.02.4 to 22.12-BETA.1 is unavailable. | Workaround is to either upgrade from a version before 22.01.4 or to upgrade to 22.12-BETA.2. | 2.12-BETA.2 |
```

Comment:
unavailable is probably the wrong word. Maybe known to not work or cause issues? @Mrt134 

Also it is 22.02.4, not 22.01.4
And 22.12-BETA.2 not 2.12-BETA.2

Reply:
Corrected typos and changed wording to "known to not work". I'll push this fix now.